packages/aragon-wrapper/src/index.js

```diff
@@ -1528,6 +1530,22 @@ export default class Aragon {
     let scriptData = script.substring(10)
     while (scriptData.length > 0) {
       const { segment, scriptLeft } = decodePathSegment(scriptData)
+      const { selector, data } = segment
+
+      if (selector === FORWARD_SIG) {
+        const forwardCallData = data.substring(10)
+        const forwardDataOffset = parseInt(`0x${forwardCallData.substring(0, 64)}`, 16) * 2
+        const forwardDataStartIndex = forwardDataOffset + 64
+        const forwardDataLength = parseInt(`0x${forwardCallData.substring(forwardDataOffset, forwardDataStartIndex)}`, 16) * 2
+        const forwardData = `0x${forwardCallData.substring(forwardDataStartIndex, forwardDataStartIndex + forwardDataLength)}`
+
+        // If forwarding data is a script decode it
+        const forwardScriptId = script.substring(0, 10)
```

Comment:
```suggestion
        const forwardScriptId = forwardData.substring(0, 10)
```

I think we should be using the forwarded script here?

Reply:
nice catch haha, it was working by chance